.github/workflows/master_build_pr.yml

```diff
@@ -545,7 +547,7 @@ jobs:
     if: ${{ inputs.type == 'amd-go-22' }}
     runs-on: amd-go-22
     env:
-      CI_BRANCH: main
+      CI_BRANCH: TDEVOPS-2780-1
```

Comment:
We can now remove every go specific builds altogether keeping only `build-amd`

Reply:
Yes.